cmd.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+import fs from 'fs';
+import {convertDocsHtmlToMarkdown} from './lib/convert.js';
+
+// to avoid top-level async/await. readFileSync(0) caused strange errors
+// https://stackoverflow.com/questions/67037353/any-way-to-read-from-stdin-synchronously-without-extra-modules-libraries-node-j
+function stdinReadSync() {
```

Comment:
Why go through the extra work here? It’s fine to avoid top-level async/await, but you can still just:

```
main()
  .then(result => process.stdout.write(result))
  .catch(error => process.stderr.write(`Unknown error: ${error});
```